Jenkinsfile

```diff
@@ -90,6 +91,7 @@ pipeline {
                 steps {
                     provisionNodes NODELIST: env.NODELIST,
                                    distro: 'el7',
+                                   profile: 'ci',
```

Comment:
Should we be future-looking here and call this something like `ci-daos` in anticipation of having other profiles for CI for other projects that need a different image?

Reply:
This is testing the new CI profile to make sure it works.
As we add more profiles to support we can convert it to a matrix and manage the names.
It will probably be a while before we add more profiles.

Reply:
Created daos_ci profile.